java/ql/src/utils/stub-generator/Stubs.qll

```diff
@@ -27,19 +28,24 @@ abstract private class GeneratedType extends ClassOrInterface {
   }
 
   private string stubStaticModifier() {
-    if this.isStatic() then result = "static " else result = ""
+    if this.(NestedType).isStatic() then result = "static " else result = ""
   }
 
   private string stubAccessibilityModifier() {
     if this.isPublic() then result = "public " else result = ""
   }
 
+  private string stubAnnotations() {
+    result = concat(stubAnnotation(this.(AnnotationType).getAnAnnotation()) + "\n")
```

Comment:
Do we need a stable order for the annotations? I see that we don't order the members in `stubMembers` either, but we do `order by` all `concat`s in C#. Just wondering.

Reply:
This would seem wise, ordering by their fully-qualified names for example